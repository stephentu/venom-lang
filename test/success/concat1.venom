a = "hi";
b = ", foo";
print (a.concat(b));
print (a + b);
print ("hello " + "world");
