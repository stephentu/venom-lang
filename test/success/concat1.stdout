hi, foo
hi, foo
hello world
